e2e/web/specs/play-sudoku.spec.ts

```diff
@@ -310,6 +318,14 @@ test.describe("erase", () => {
     await sudokuBoard.cell[7][6].click();
     await sudokuBoard.erase.click();
     await sudokuBoard.cellHasValue(7, 6, "0");
+    // Use keyboard shortcuts to erase the value
+    const keys = ["Delete", "Backspace", "0", "e", "E"];
+    for (const key of keys) {
+      await sudokuBoard.undo.click();
+      await sudokuBoard.cellHasValue(7, 6, "1");
+      await sudokuBoard.cell[7][6].press(key);
+      await sudokuBoard.cellHasValue(7, 6, "0");
+    }
```

Comment:
Could we update the test name to show that it is testing for all erase methods? Maybe "Erasing an incorrect value should succeed for all erase paths"?

Reply:
Done!